CharacterCreator/CharacterCreator/Models/Race.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CharacterCreator.Models
+{
+    public class Race
```

Comment:
Since this is providing a value to another table (Characters), we need to configure it as such. Since a single race can apply to multiple characters, we need to declare a collection of Characters to Race. In other words, include:

`public List<Character> Characters {get; set;}`

at the very bottom to achieve this

Reply:
Fixed